go.mod

```diff
@@ -123,21 +124,35 @@ require (
 	nhooyr.io/websocket v1.8.7 // indirect
 )
 
+// the sdk's mandatory replaces
 replace (
-	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
-	github.com/aws/aws-sdk-go v1.25.48 => github.com/aws/aws-sdk-go v1.33.0
-	github.com/aws/aws-sdk-go v1.27.0 => github.com/aws/aws-sdk-go v1.33.0
-	github.com/cosmos/cosmos-sdk => github.com/classic-terra/cosmos-sdk v0.45.13-classic
-	github.com/cosmos/ledger-cosmos-go => github.com/terra-money/ledger-terra-go v0.11.2
-	github.com/docker/docker v1.4.2-0.20180625184442-8e610b2b55bf => github.com/docker/docker v1.6.1
-	github.com/ethereum/go-ethereum v1.9.25 => github.com/ethereum/go-ethereum v1.10.9
-	github.com/gin-gonic/gin v1.4.0 => github.com/gin-gonic/gin v1.7.7
+	// use cosmos fork of keyring
+	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
+	// dgrijalva/jwt-go is deprecated and doesn't receive security updates.
+	// TODO: remove it: https://github.com/cosmos/cosmos-sdk/issues/13134
+	github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt/v4 v4.4.2
+
+	// Fix upstream GHSA-h395-qcrw-5vmq vulnerability.
+	// TODO Remove it: https://github.com/cosmos/cosmos-sdk/issues/10409
+	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.8.1
+
+	// Use regen gogoproto fork
+	// This for is replaced by cosmos/gogoproto in future versions
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-	github.com/libp2p/go-buffer-pool v0.0.2 => github.com/libp2p/go-buffer-pool v0.1.0
-	github.com/microcosm-cc/bluemonday v1.0.2 => github.com/microcosm-cc/bluemonday v1.0.16
-	github.com/miekg/dns v1.0.14 => github.com/miekg/dns v1.1.25
-	github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v1.2.0
```

Comment:
just bump tm-db and you'll never need this again

Reply:
Long term we might choose to fork TM-DB derived by this initial performance optimization work @ https://github.com/terra-money/tm-db/tree/v0.6.7-performance.4 and hopefully finding time to implement logic to allow "pass-thru-configuration" for the underlying DBMS provider implementations (GoLevelDb, BadgerDb, RocksDb, etc) so we can actually enable validators to tweak their configurations a bit more then what is currently possible in TM-DB